SUITesslaApp/SUITesslaApp/SettingsScreen/SettingsScreenView.swift

```diff
@@ -0,0 +1,542 @@
+
+    var body: some View {
+        ZStack {
+            Constants.backgroundTopColor
+                .ignoresSafeArea()
+            VStack {
+                headBarView
+                settingsValueView
+                settingsDisclosureGroupView.padding()
+                Spacer()
+            }
+            activateSystemBottomSheetView
+        }
+    }
+
+    // MARK: - Private properties
+
+    @GestureState private var gestureOffset = CGSize.zero
+    @State private var currentMenuOffsetY: CGFloat = 0.0
+    @State private var lastMenuOffsetY: CGFloat = 0.0
+    @State private var isCarClosed = false
+    @State private var tagSelected = 0
+    @State private var acSliderValue: Float = 0.0
+    @State private var shadowRadius = 0.0
+    @State private var completionAmout = 0.0
+    @State private var selectedColor = Color.white
+    @State private var temp = 15.0
+
+    private var goPreviusScreenButtonView: some View {
+        ZStack {
+            Circle()
+                .fill(Constants.backgroundTopColor)
+                .shadow(color: .gray, radius: 5, x: -3, y: -3)
+                .shadow(color: .black, radius: 5, x: 5, y: 5)
+                .frame(width: 73, height: 73)
+            Button {
+
+            } label: {
+                Image(systemName: Constants.goPreviusButtonImageName)
+                    .renderingMode(.template)
+                    .foregroundColor(.gray)
+                    .frame(width: 50, height: 50)
+                    .settingsNeumorfismSelectedCircleStyle()
+            }
+        }
+    }
+
+    private var settingsButtonView: some View {
+        ZStack {
+            Circle()
+                .fill(Constants.backgroundTopColor)
+                .shadow(color: .gray, radius: 5, x: -3, y: -3)
+                .shadow(color: .black, radius: 5, x: 5, y: 5)
+                .frame(width: 73, height: 73)
+            Button {
+
+            } label: {
+                Image(systemName: Constants.settingsGearshapeImageName)
+                    .renderingMode(.template)
+                    .foregroundColor(.gray)
+                    .frame(width: 50, height: 50)
+                    .settingsNeumorfismUnSelectedCircleStyle()
+            }
+        }
+    }
+
+    private var climateTextView: some View {
+        Text(Constants.climateTextViewText)
+            .padding()
+            .foregroundColor(.white)
+    }
+
+    private var headBarView: some View {
+        HStack {
+            goPreviusScreenButtonView
+            climateTextView
+                .padding(EdgeInsets(top: 0, leading: 50, bottom: 0, trailing: 50))
+            settingsButtonView
+        }
+    }
+
+
+    private var settingsValueView: some View {
+        ZStack {
+            Circle()
+                .fill(LinearGradient(colors: [.black, .gray], startPoint: .topLeading, endPoint: .bottomTrailing))
+                .frame(width: 151, height: 151)
+            Circle()
+                .stroke( // 1
```

Comment:
```suggestion
                .stroke( 
```
и ниже комменты поубирай